.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+<!-- Please fill out any relevant sections and remove those that don't apply -->
+## Motivation
+<!-- Why was this pull request needed? -->
+## Description of changes
+<!-- What was changed in this pull request? -->
+## Breaking changes
+<!-- What breaking changes were added (if any) and how are they addressed? -->
+## Additional checks
+<!-- Please check what applies. Note that some of these are not hard requirements but merely serve as information for reviewers. -->
+- [ ] Updated the documentation of the project to reflect the changes
+- [ ] Added new tests that cover the code changes
+- [ ] Fits CONTRIBUTING.md
```

Comment:
I think our CONTRIBUTING,md is always "we do not accept" external contribution so not much to check here ^^. We could remove.

Reply:
As discussed in vc, we will remove this.